service-providers/ip-packet-router/src/constants.rs

```diff
@@ -2,8 +2,10 @@ use std::time::Duration;
 
 // The interface used to route traffic
 pub const TUN_BASE_NAME: &str = "nymtun";
-pub const TUN_DEVICE_ADDRESS: &str = "10.0.0.1";
-pub const TUN_DEVICE_NETMASK: &str = "255.255.255.0";
+pub const TUN_DEVICE_ADDRESS_V4: &str = "10.0.0.1";
```

Comment:
would it perhaps make more sense to define those using concrete types? 
i.e. for example
```rust
pub const TUN_DEVICE_ADDRESS_V4: IpAddr = IpAddr::V4(Ipv4Addr::new(10,0,0,1));
```

this way you won't have to parse and unwrap anything

Reply:
Nice catch, I'll do that, thanks.